README.md

```diff
@@ -7,105 +7,98 @@
+  departments within the Municipality of Amsterdam.
 
-- Attractive for the (internal) open source community; could even be used outside Datapunt in other departments within the Municipality of Amsterdam.
+### Risks
 
-### CONS (& risks)
+- Quality assurance; we need to set up some well founded contribution guidelines and make sure the
+  repo doesn’t get polluted.
+- Versioning: updating the component package might contain breaking changes. This could be prevented
+  for most of the time if we have a proper updating strategy in our guidelines.
+- More time to set up, develop and maintain. However, this is an investment for future products that
+  will result in decreasing development time drastically.
 
-- Quality assurance; we need to set up some well founded contribution guidelines and make sure the repo doesn’t get polluted.
+### Contributing
 
-- Versioning: updating the component package might contain breaking changes. This could be prevented for most of the time if we have a proper updating strategy in our guidelines.
+Please read the [CONTRIBUTION.md](./docs/CONTRIBUTING.md)
 
-- More time to set up, develop and maintain. However, this is an investment for future products that will result in decreasing development time drastically.
 
 ## Usage
 
-### Development
-
-```bash
-yarn
-```
+Make sure you have the right node version set (see the package.json). Run `yarn` to install the
+dependencies. Next run `yarn start` to start storybook!
 
-To start watching files and start storybook, run `yarn start`
+### While developing
 
-To test your components in other repo's, do the following:
+Something you want to see your changes immediately in your project. Follow these steps to link asc
+to your project.
 
-1. Run `yarn build:watch`, this will watch your files and transpile them to the package lib directory.
+1. Run `yarn build:watch`, this will watch your files and transpile them to the package lib 
+   directory.
 2. cd to the package you want to test, run `yarn link`.
-3. Go to the repo where you want to use your package and run `yarn link <package-name>`.`<package-name>` can be found in the `package.json` you linked in step 2.
-
-Now you can import the package like you would do like a normal npm dependency. Changes you will make in your package will be seen in your repo.
-
-- There is a known issue when developing with `yarn link` and using Hooks [click her for details](https://reactjs.org/warnings/invalid-hook-call-warning.html). The cause is that 2 React instances are used when using link. 
-- To solve this problem for the `./examples/create-react-app`, run
-```bash
-npm link ./examples/create-react-app/node-modules/react
-```
-from the root folder to use the same React version in the component lib as in the application.
-
-#### Generate components from templates
+3. Go to the repo where you want to use your package and run 
+   `yarn link <package-name>`.`<package-name>` can be found in the `package.json` you linked in 
```

Comment:
`yarn link <package-name>`. The`<package-name>`......
Add `The ` including a space to make it more readable